eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/SegmentNr.scala

```diff
@@ -3,18 +3,90 @@ package com.evolutiongaming.kafka.journal.eventual.cassandra
 import cats.kernel.Eq
 import cats.syntax.all.*
 import cats.{Applicative, Id, Monad, Order, Show}
-import com.evolutiongaming.kafka.journal.SeqNr
 import com.evolutiongaming.kafka.journal.util.Fail
 import com.evolutiongaming.kafka.journal.util.Fail.implicits.*
+import com.evolutiongaming.kafka.journal.{Key, SeqNr}
 import com.evolutiongaming.scassandra.{DecodeByName, DecodeRow, EncodeByName, EncodeRow}
 
+/**
+  * Segment fields in `journal` and `metajournal` tables. Part of clustering key in both tables.
+  * 
+  * [[SegmentNr]] can be limited: in `metajournal` table cannot be more than [[Segments]] thus:
+  * ```
+  *    val metaJournalSegmentNr: SegmentNr = ???
+  *    val metaJournalSegments: Segments = ???
+  *    metaJournalSegmentNr <= metaJournalSegments // have to be true
+  * ```
+  * 
+  * In `journal` table [[SegmentNr]] is not limited but rather each segment contains not more than [[SegmentSize]] events:
+  * ```
+  *   val journalSegmentNr: SegmentNr = ???
+  *   val journalSegmentSize: SegmentSize = ???
+  *   journalSegmentNr.value > journalSegmentSize.value // can be true but not have to be
+  * ```
```

Comment:
this is misleading - the inequality doesn't make sense as there is no dependency between each other.
```suggestion
```

Reply:
this is not misleading but exact example of values that can have `SegmentNr` and `SegmentSize`